src/test/scala/com/fulcrumgenomics/bam/pileup/PileupBuilderTest.scala

```diff
@@ -289,6 +289,26 @@ class PileupBuilderTest extends UnitSpec {
       piler.safelyClose()
     }
 
+    it should "not filter out single-end records when we are removing records outside the insert of FR pairs" in {
```

Comment:
```suggestion
    it should "not filter out single-end records when mappedPairsOnly is false" in {
```

Reply:
I updated the test name to be more clear about that!